<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterSelection</class>
 <widget class="QMainWindow" name="ParameterSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameter Selection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="mainGridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QTreeWidget" name="treeWidget">
        <attribute name="headerDefaultSectionSize">
         <number>90</number>
        </attribute>
        <column>
         <property name="text">
          <string>Layer Name</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="applyPushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="toggleselectAllLayersPushButton">
        <property name="text">
         <string>Select All Layers</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>applyPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
